eslint-ts.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    "extends": "maven",
+    "env": {
+        "node": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "plugins": ["@typescript-eslint"],
```

Comment:
What do you think about moving the dependency to these packages into this repo, assuming there is no negative impact on other node projects?

Reply:
Good idea, done